themes/theme-gmd/components/ProductCard/index.jsx

```diff
@@ -59,8 +59,14 @@ const ProductCard = ({
           </div>
         )}
 
+        {/* This feature is currently in BETA testing.
+          It should only be used for approved BETA Client Projects */}
         <MapPriceHint productId={product.id} />
 
+        {/* This feature is currently in BETA testing.
+          It should only be used for approved BETA Client Projects */}
+        <OrderQuantityHint productId={product.id} />
```

Comment:
`OrderQuantityHint` and `MapPriceHint` cause `No page config found for page pattern "/"` error logs, when the `ProductCard` component is used on pages where no widget settings exist for. That's the case e.g. on an `index` page with product sliders.

Same for the ios theme.